src/main/java/org/folio/rest/persist/HelperUtils.java

```diff
@@ -146,4 +148,10 @@ public static List<Error> buildNullValidationError(String value, String key) {
     return Collections.emptyList();
   }
 
+  public static <T, R> Future<R> chainCall(List<T> list, Function<T, Future<R>> method){
```

Comment:
add a javadoc explaining what this method does.

Reply:
Added